src/libcore/macros.rs

```diff
@@ -8,6 +8,16 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+#[macro_export]
+// This stability attribute is totally useless.
+#[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
Oh OK, missed the `#[cfg(stage0)]`. So it will never even reach rustdoc. Carry on then.